### ISSUE TITLE
Feature Request: Overwrite $lang['none'] by an individual message

### ISSUE THREAD

**Issue by dranjor** [cosmocode/dokuwiki-plugin-struct#525]
By default struct returns 'Nothing found' if nothing has matched. It would be nice, if one could overwrite this parameter-driven by an individual message:

---- struct table ----
schema: schema_name
cols: %pageid%, other, cols
notfoundmsg: "Sorry, not data available. Please try [[page]]."
----

**Comment by annda** [comment]
You can already customize any plugin messages yourself by using the localization mechanism: https://www.dokuwiki.org/localization?s[]=plugin_lang#changing_some_localized_texts_and_strings_in_your_installation

What will not work is wiki syntax in the string, so the link in your example would not be parsed, but instead rendered as is.

**Comment by dranjor** [comment]
You are right. But I want in different aggregations different messages. :-)
